action.yml

```diff
@@ -0,0 +1,49 @@
+name: Deploy AWS Lambda
+description: Bundles a JS Lambda function and deploys it to AWS
+inputs:
+  lambda-dir:
+    description: Directory of the Lambda
+    required: true
+  lambda-name:
+    description: Name of the Lambda
+    required: true
+  aws-region:
+    description: AWS Region
+    default: eu-central-1
+    required: false
+  aws-access-key-id:
+    description: AWS Access Key ID
+    required: true
+  aws-secret-access-key:
+    description: AWS Secret Access Key
+    required: true
+  s3-bucket:
+    description: Name of the S3 bucket
+    required: true
+  s3-key:
+    description: Key of the S3 bucket object
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-region: ${{ inputs.aws-region }}
+        aws-access-key-id: ${{ inputs.aws-access-key-id }}
+        aws-secret-access-key: ${{ inputs.aws-secret-access-key }}
+    - name: zip
+      run: |
+        cd ${{ inputs.lambda-dir }}
+        zip -qq -r ./bundle.zip ./
+      shell: bash
+    - name: S3 copy
+      run: |
+        cd ${{ inputs.lambda-dir }}
+        aws s3 cp ./bundle.zip s3://${{ inputs.s3-bucket }}/${{ inputs.s3-key }}
+      shell: bash
+    - name: update Lambda
+      run: |
+        aws lambda update-function-code --function-name ${{ inputs.lambda-name }} \
```

Comment:
Does `aws lambda update-function-code` cause the running version to update?

Reply:
I assumed that, let me verify.

Reply:
In short, yes. As we are not using Lambda versioning, there is only one version that is running (`$Latest`), which is being updated.